test/test_onboard_service.sh

```diff
@@ -4,9 +4,24 @@
 source test/utils.sh
 
 KEPTN_EXAMPLES_BRANCH=${KEPTN_EXAMPLES_BRANCH:-"master"}
+KEPTN_NAMESPACE=${KEPTN_NAMESPACE:-keptn}
+KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n "$KEPTN_NAMESPACE" -o jsonpath='{.data.keptn-api-token}' | base64 --decode)
 
 echo "Testing onboarding for project $PROJECT"
 
+echo "Using remote execution plane services: ${REMOTE_EXECUTION_PLANE}"
+
+# check if REMOTE_EXECUTION_PLANE is set to true. If yes, scale down the helm-service and jmeter in the keptn namespace and install the services via their helm charts
+if [[ "${REMOTE_EXECUTION_PLANE}" == "true" ]]; then
+  kubectl scale deployment/helm-service -n ${KEPTN_NAMESPACE} --replicas=0
+  kubectl scale deployment/jmeter-service -n ${KEPTN_NAMESPACE} --replicas=0
+
+  KEPTN_API_HOSTNAME=$(kubectl -n ${KEPTN_NAMESPACE} get service api-gateway-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  KEPTN_VERSION_TAG=$(kubectl get deployment -n ${KEPTN_NAMESPACE} helm-service -ogo-template='{{index .metadata.labels "app.kubernetes.io/version"}}')
```

Comment:
Fwiw, you might be able to add
```
 -o
```
to your `patterns.txt` file instead of adding `ogo` to your `expect.txt` file